bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/TimeAmount.scala

```diff
@@ -13,7 +16,9 @@ object TimeAmount {
   // with the difference that we try to keep the time unit from changing if they
   // are all the same.
   def sum(times: Traversable[TimeAmount]): TimeAmount =
-    if (times.isEmpty) empty else (times.head/:times.tail)(_ + _)
+    if (times.isEmpty) empty else (times.head/:times.tail)(_ |+| _)
```

Comment:
Yeah it's not going to work with `Traversable`. But with `Foldable` this operation is just `subs.foldMap(f)` and I'll bet all the callers are `Foldable`. I didn't know this stuff when I wrote the PIT.

Not a big deal if you don't feel like chasing this down.